packages/reactivue/src/component.ts

```diff
@@ -4,13 +4,26 @@ import { isDev } from './env'
 import { invokeLifeCycle } from './lifecycle'
 import { InternalInstanceState, LifecycleHooks } from './types'
 
-let _id = 0
-const _vueState: Record<number, InternalInstanceState> = {}
+type ReactiveEffects = Record<number, InternalInstanceState>
+
+declare global {
+  interface Window { _reactivueState: ReactiveEffects }
+}
+
+const _vueState: ReactiveEffects = (isDev && window && window._reactivueState) || {}
```

Comment:
`window` need to be checked by `typeof window !== 'undefined`, you can save it as `isClient`